### PR TITLE
Show search input in small screens #842

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #842 
Include search input when screen is small

### CODE REVIEW

**Comment by jlprat** [comment]
How it would look in big screens:
search_big
How it would look in mid screens:
search_mid
How it would look in small screens:
search_small

File: docs/src/main/paradox/_template/page.st [hunk 1]
```diff
@@ -39,13 +39,17 @@
           <a href="#" class="off-canvas-toggle hide-for-medium" data-toggle="off-canvas-menu">$menu()$</a>
 
           <img src="$page.base$css/../mini-akka-logo.png" alt="Akka logo" style="float: left"/> $title()$
+
+          <div style="position: relative; right:0; min-width:25%" class="float-right">
+            <input type="search" id="search" class="form-control" style="margin: 10px 0;" placeholder="Search the docs..."/>
+          </div>
+
```
Comment (jonas) [inline docs/src/main/paradox/_template/page.st]: I think code in the footer be removed then:

```
$! Override Algolia's search wrapper styles to let the search box span the whole left panel !$
 <style>.algolia-autocomplete { display: block !important }</style>
```

Comment (jlprat) [inline docs/src/main/paradox/_template/page.st]: Checking it right now

Comment (jlprat) [inline docs/src/main/paradox/_template/page.st]: If this is removed the input field is smaller then...

Comment (jonas) [inline docs/src/main/paradox/_template/page.st]: OK, thanks for checking

**Pull request merged**